### PR TITLE
Shorter BTT board names (for LCD info)

### PR DESCRIPTION
**Pull request by Lord-Quake**
Make sure the complete string is visible on LCD screen and version numbers are not cut off.
Change BIGTREETECH to BTT to shorten string and have a consistent convention for all BIGTREETECH board names.

### CODE REVIEW

**Comment by thisiskeithb** [comment]
I had fixed the mini boards in my PR, but I can drop that commit: https://github.com/MarlinFirmware/Marlin/commit/813a61ce0b504bef3d61cf61cc598e15113ab0b3

**Comment by Lord-Quake** [comment]
When I looked this morning it wasn't there :-(
If you are happy with my changes then we can leave them otherwise let me know if I should drop mine.

File: Marlin/src/pins/stm32f1/pins_BTT_SKR_E3_DIP.h [hunk 1]
```diff
@@ -25,7 +25,7 @@
   #error "Oops! Select an STM32F1 board in 'Tools > Board.'"
 #endif
 
-#define BOARD_INFO_NAME "BIGTREE SKR E3 DIP V1.0"
+#define BOARD_INFO_NAME "BTT SKR E3 DIP V1.0"
```
Comment (thisiskeithb) [inline Marlin/src/pins/stm32f1/pins_BTT_SKR_E3_DIP.h:28]: For consistency with the other E3 Mini boards, I changed this to "BTT SKR Mini E3 DIP V1.0". While longer, it's still under the 25 character limit.

File: Marlin/src/pins/stm32f1/pins_BTT_SKR_MINI_E3_V1_0.h [hunk 2]
```diff
@@ -23,7 +23,7 @@
 
 #include "pins_BTT_SKR_MINI_E3.h"
 
-#define BOARD_INFO_NAME "BIGTREE SKR Mini E3"
+#define BOARD_INFO_NAME "BTT SKR Mini E3"
```
Comment (thisiskeithb) [inline Marlin/src/pins/stm32f1/pins_BTT_SKR_MINI_E3_V1_0.h]: For consistency with the other E3 Mini boards, I changed this to "BTT SKR Mini E3 V1.0". This comes in right under the 20 character limit.

**Comment by thisiskeithb** [comment]
I removed that commit from my PR and left comments above to update this one 🙂

File: Marlin/src/pins/stm32f4/pins_BTT_BTT002_V1_0.h [hunk 3]
```diff
@@ -27,7 +27,7 @@
   #error "BIGTREE BTT002 V1.0 supports up to 1 hotends / E-steppers."
 #endif
 
-#define BOARD_INFO_NAME "BIGTREE Btt002 1.0"
+#define BOARD_INFO_NAME "BTT Btt002 V1.0"
```
Comment (thisiskeithb) [inline Marlin/src/pins/stm32f4/pins_BTT_BTT002_V1_0.h]: Since we're already changing names... let's capitalize BTT002: "BTT BTT002 V1.0"

**Comment by Lord-Quake** [comment]
OK, do I have to do anything further?

**Comment by thisiskeithb** [comment]
> OK, do I have to do anything further?

See my suggestions above. Edit: "BTT SKR Mini E3 DIP V1.0" was still over the 20 character limit, so I don't think that one can be formatted like the others by adding in "Mini".

**Comment by Lord-Quake** [comment]
OK, so I have to make the changes locally and then push the new files up again.

Will get to it after my MS Teams session is over :-)

**Comment by Lord-Quake** [comment]
@thisiskeithb Because I'm new to this.... Do I have to delete this pull request and repeat it again because the files with the latest bugfix fork have been updated or can I leave this pull requests as is?

**Comment by dantob** [comment]
what about the lpc1768 and lpc1769 based BIGTREETECH boards for consistency?

**Comment by Lord-Quake** [comment]
Yep, I missed those. Will update them as well.

**Comment by Lord-Quake** [comment]
@thisiskeithb What do you think? It'd be nice if we could go for a commit.

**Comment by thisiskeithb** [comment]
Looks fine, but merging PRs isn’t up to me 🙂

**Comment by Lord-Quake** [comment]
Well, at least I feel better now knowing a pro like yourself feels positive about this commit 🙂

**Pull request merged**